### DISCUSSION TITLE
Get the list of emoji shortcodes supported by the API

### DISCUSSION THREAD

**Discussion by kevincador** [trakt/trakt-api#347] [Q&A] [answered]
Hi,

Would it be possible to get the list of shortcodes `:smiley:` supported by trakt (via the API?)?

I would like to use this list to let the user know a comment won't look exactly like what he typed because some emojis are going to be stripped from it.

Thanks!

**Comment by rectifyer** [comment, marked as answer]
We use JoyPixels to render the emoji so here are a few resources:

https://trakt.docs.apiary.io/#introduction/emojis
https://www.joypixels.com/emoji
https://www.joypixels.com/fonts
https://github.com/joypixels/emoji-toolkit

That GitHub project should have source code with the shortcodes and what they translate to. We use that toolkit on the website.
